conversation/webhook.go

```diff
@@ -0,0 +1,50 @@
+package conversation
+
+import (
+	"net/http"
+
+	messagebird "github.com/messagebird/go-rest-api"
+)
+
+type WebhookCreateRequest struct {
+	ChannelID string         `json:"channelId"`
+	Events    []WebhookEvent `json:"events"`
+	URL       string         `json:"url"`
+}
+
+// CreateWebhook registers a webhook that is invoked when something interesting
+// happens.
+func CreateWebhook(c *messagebird.Client, req *WebhookCreateRequest) (*Webhook, error) {
+	webhook := &Webhook{}
+	if err := request(c, webhook, http.MethodPost, webhooksPath, req); err != nil {
+		return nil, err
+	}
+
+	return webhook, nil
+}
+
+// DeleteWebhook ensures an existing webhook is deleted and no longer
+// triggered. If the error is nil, the deletion was successful.
+func DeleteWebhook(c *messagebird.Client, id string) error {
+	return request(c, nil, http.MethodDelete, webhooksPath+"/"+id, nil)
+}
+
+// ListWebhooks gets a collection of webhooks. Pagination can be set in options.
+func ListWebhooks(c *messagebird.Client, options *ListOptions) (*WebhookList, error) {
+	webhookList := &WebhookList{}
+	if err := request(c, webhookList, http.MethodGet, webhooksPath, options); err != nil {
```

Comment:
Self review: add query parameters to the path. The value for `data` is sent as a JSON encoded body, which doesn't work with GET.

Reply:
Resolved in https://github.com/messagebird/go-rest-api/pull/49/commits/430d19fb45c9cf0ee346a0d76a3b611ca78cf2be